scripts/quests/windurst/Creepy_Crawlies.lua

```diff
@@ -0,0 +1,103 @@
+-----------------------------------
+-- Creepy Crawlies
+-----------------------------------
+-- Log ID: 2, Quest ID: 39
+-- Illu Bohjaa : !pos 21.35 -5 125 241
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CREEPY_CRAWLIES)
+
+quest.reward =
+{
+    title = xi.title.CRAWLER_CULLER,
+    famre = 30,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Illu_Bohjaa'] = quest:event(333, 0, xi.items.SPOOL_OF_SILK_THREAD, xi.items.PAPAKA_GRASS, xi.items.CRAWLER_CALCULUS),
+
+            onEventFinish =
+            {
+                [333] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Illu_Bohjaa'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if
+                        npcUtil.tradeHas(trade, { { xi.items.SPOOL_OF_SILK_THREAD, 3 } }) or
+                        npcUtil.tradeHas(trade, { { xi.items.CRAWLER_CALCULUS, 3 } })
+                    then
+                        return quest:event(335, 600, xi.items.SPOOL_OF_SILK_THREAD, 0, xi.items.CRAWLER_CALCULUS)
```

Comment:
600 * xi.settings.main.GIL_RATE, like in the addGil below

Reply:
not rewarding gil at this point. the cs text shows 600 is the params.

Reply:
It shows 600 in the CS, but you'll be rewarding 600 * GIL_RATE, no?

Reply:
ie. the default is 1.0, so if you don't fiddle with the settings it'll be retail accurate, but if you change the settings the CS won't reflect the change - and we have the capability to reflect the change, so why wouldn't we?